detox/local-cli/testCommand/TestRunnerCommand.js

```diff
@@ -0,0 +1,173 @@
+const cp = require('child_process');
+
+const _ = require('lodash');
+const parser = require('yargs-parser');
+const unparse = require('yargs-unparser');
+
+const detox = require('../../internals');
+const log = detox.log.child({ cat: ['lifecycle', 'cli'] });
+const { DetoxRuntimeError } = require('../../src/errors');
+const { printEnvironmentVariables, prependNodeModulesBinToPATH } = require('../../src/utils/envUtils');
+const { escapeSpaces } = require('../../src/utils/shellUtils');
+
+class TestRunnerCommand {
```

Comment:
Missing unit tests

Reply:
I don't plan to cover it with unit tests – spawning the child processes is a too-specific domain.

We have `local-cli/test.test.js`, which covers all the cases (except `--inspect-brk`, but I plan to resurrect that soon).